docs/read-the-docs/source/tls.rst

```diff
@@ -0,0 +1,83 @@
+TLS support
+===========
+
+Receptor supports mutual TLS authentication and encryption for above and below the
+mesh connections.
+
+Configuring TLS
+^^^^^^^^^^^^^^^
+
+Add ``tls-server`` and ``tls-client`` definitions to receptor config files.
+
+foo.yml
+
+.. code-block:: yaml
+
+    ---
+    - node:
+        id: foo
+
+    - log-level:
+        level: Debug
+
+    - tls-server:
+        name: myserver
+        cert: foo.crt
+        key: foo.key
+        requireclientcert: true
+        clientcas: ca.crt
+
+    - tcp-listener:
+        port: 2222
+        tls: myserver
+
+Having ``tls-server`` defined does nothing on its own, but it can be referenced elsewhere in the receptor config file. Above, ``tls`` in the ``tcp-listener`` is set to use ``myserver``. In general, ``tls-server`` should be referenced any where receptor is expecting an incoming connection, i.e. ``*-listener`` backends or on the ``control-service``. Similarly, ``tls-client`` should be referenced any where receptor is expecting to make an outgoing connection, i.e. ``*-peer`` backends or in receptorctl (the command-line client for receptor).
+
+bar.yml
+
+.. code-block:: yaml
+
+    ---
+    - node:
+        id: bar
+
+    - log-level:
+        level: Debug
+
+    - tls-client:
+        name: myclient
+        rootcas: ca.crt
```

Comment:
would this just look for the file in the local directory, or does receptor search canonical folders used to hold certs on the system?